README.md

```diff
@@ -92,8 +92,8 @@ var contextFactory = ContextFactory.Instance;
 var monitor = new SCardMonitor(contextFactory, SCardScope.System);
 // connect events here..
 
-/* Attention: this is an example, please do not use anonymous methods
- * as event handlers to avoid memory leaks. */
+/* Attention: In order to avoid memory leaks, do not use anonymous methods

```

Comment:
Using a lambda for an event handler does not cause a memory leak, it just makes it difficult to disconnect the handler. If there's no need to disconnect, then a lambda is fine; it will be garbage collected when the monitor is collected.

I would suggest removing this entire comment. I think a developer who wishes to disconnect the event handler can be expected to understand how to code for it correctly.